portal-backend/tests/depmap/data_page/test_api.py

```diff
@@ -119,15 +268,55 @@ def mock_get_all_data_avail_df():
                 "Drug_CTD_Broad": 3,
                 "Drug_Repurposing_Broad": None,
             },
+            "lineage_counts": {
+                "CRISPR_Achilles_Broad": {
+                    "Bone": {"Ewing Sarcoma": "2", "Osteosarcoma": "2"},
+                    "Lung": {},
+                    "Myeloid": {"Acute Myeloid Leukemia": "1"},
+                },
+                "Drug_CTD_Broad": {
+                    "Bone": {"Osteosarcoma": "2", "Ewing Sarcoma": "1"},
+                    "Lung": {},
+                    "Myeloid": {"Acute Myeloid Leukemia": "2"},
+                },
+                "Drug_Repurposing_Broad": {
+                    "Bone": {"Ewing Sarcoma": "2", "Osteosarcoma": "1"},
+                    # Is it possible for a lineage to not have primary diseases?
+                    # Or are the primary disease counts always going to add up to the
+                    # lineage count?
+                    "Lung": {},
+                    "Myeloid": {"Acute Myeloid Leukemia": "1"},
```

Comment:
@pgm I'm not quite done with this yet, which is why I still have it as a draft PR, but I realized when I was building this test, I'm not sure whether or not I can make this assumption: Is it possible for a model to have an oncotree lineage but no primary disease's? Or is the data I created for this test (Lung lines with no data for primary disease) impossible?